homeassistant/components/shelly/sensor.py

```diff
@@ -153,6 +153,7 @@ class RestSensorDescription(RestEntityDescription, SensorEntityDescription):
         suggested_display_precision=2,
         device_class=SensorDeviceClass.POWER_FACTOR,
         state_class=SensorStateClass.MEASUREMENT,
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
Aren't these just other measurements of the electricity the device is measuring? Ie, not a diagnostic of the device itself.

Reply:
They are used to calculate the actual Watt being used.

Reply:
The Watt used in the Home, not used by the device itself , right?

Reply:
Although I would like to see less entities on the dashboard, marking these entities as diagnostics is incorrect. 

Shelly Pro 3EM is a energy & power meter, being a power meter it reports power usage with multiple sensors.
I never used the power factor or reactive power, however people with solar panel do use them.

These are measurements that the device measures and not internal measurements so they are not diagnostics.

I highly support reducing the number of dashboard entities for such usage, but I don't think marking them as diagnostic is the way. 

I think this should be covered by either:
- A new category for marking primary/secondary entities, so that secondary won't show on the dashboard.
- https://github.com/home-assistant/architecture/discussions/957 which may help grouping entities by channel for such devices so that each channel is shown in a separate panel.

Side note: We already disabled some of the sensors for this device for example:
https://github.com/home-assistant/core/blob/9aca311451808164eedcd78746c4336f925f6377/homeassistant/components/shelly/sensor.py#L798-L806
which made people complain that sensors that they can see on the device web are not available on HA (it is not easy to understand they are disabled and enable them). Since we try to make HA represent the device we should do it better and not disable or hide sensors we think are less used if they are part of what the device reports.